sdk/README.md

```diff
@@ -18,33 +17,67 @@ a basic sequential pipeline.
 
 ## Steps
 
-1. Setup Python environment with Conda or a Python virtual environment:
+1. Clone the kfp-tekton repo:
+    - `git clone https://github.com/kubeflow/kfp-tekton.git`
+    - `cd kfp-tekton`
+
+
+2. Setup Python environment with Conda or a Python virtual environment:
 
     - `python3 -m venv .venv`
     - `source .venv/bin/activate`
 
-2. Build the compiler:
+3. Build the compiler:
 
     - `pip install -e sdk/python`
 
-3. Run the compiler tests (optional):
+4. Run the compiler tests (optional):
 
     - `./sdk/python/tests/run_tests.sh`
 
-4. Compile the sample pipeline:
+5. Compile the sample pipeline:
 
     - `cd sdk/samples`  
     - `dsl-compile-tekton --py parallel_join.py --output pipeline.yaml`
     
-5. Run the sample pipeline on a Tekton cluster:
+6. Run the sample pipeline on a Tekton cluster:
 
     - `kubectl apply -f pipeline.yaml`
     - `tkn pipeline start parallel-pipeline`
 
+   You should see messages asking for default URLs like below. Press `enter` and take the defaults
+    ```bash
+      ? Value for param `url1` of type `string`? (Default is `gs://ml-pipeline-playgro 
+      ? Value for param `url1` of type `string`? (Default  is `gs://ml-pipeline-playground/shakespeare1.txt`) gs://ml-pipeline-
+      playground/shakespeare1.txt
+      ? Value for param `url2` of type `string`? (Default is `gs://ml-pipeline-playgro? Value for param `url2` of type `string`? (Default 
+      is  `gs://ml-pipeline-playground/shakespeare2.txt`) gs://ml-pipeline-playground/shakespeare2.txt
+ 
+      Pipelinerun started: parallel-pipeline-run-th4x6
+      
+      In order to track the pipelinerun progress run:
+      tkn pipelinerun logs parallel-pipeline-run-th4x6 -f -n default
+    ```
+ 7.  Run the command output in previous step to track the logs of the running Tekton Pipeline.
+
+      ```bash
+      tkn pipelinerun logs parallel-pipeline-run-th4x6 -f -n default
```

Comment:
a more general command that can actually be copy pasted from this README would be:

`tkn pipeline logs parallel-pipeline --last`

Reply:
otherwise:
`-n default`  should probably be 
`-n tekton-pipelines` which is where the Tekton Pipelines controller installed by default

Reply:
we can skip step 7 altogether if we change the `tkn` cmd in step 6:
-  `tkn pipeline start parallel-pipeline`
to
- `tkn pipeline start parallel-pipeline  --showlog`

which will show logs right after starting the pipeline.